### PR TITLE
rename users

### PR DESCRIPTION
**Pull request by will-moore**
This sets firstName and lastName for user-1 to user-40.
Uses a list of scientist names (mostly biologists/medicine), some more famous than others.
Equal number of male and female names.
Any vetoes or name change suggestions?
Haven't run this against outreach yet, but can do.

cc @pwalczysko @jburel

### CODE REVIEW

File: maintenance/scripts/rename_users.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            username = 'user-%s' % (i + 1)
+            print username, full_name
+            exp = admin_service.lookupExperimenter(username)
+            names = full_name.split(" ")
+            exp.firstName = rstring(names[0])
+            exp.lastName = rstring(names[1])
+            admin_service.updateExperimenter(exp)
+
+    except Exception as exc:
+            print "Error while renameing users: %s" % str(exc)
```
Comment (jburel) [inline maintenance/scripts/rename_users.py]: renaming

**Comment by pwalczysko** [comment]
The list of names is great, pretty impressive, thank you

**Comment by pwalczysko** [comment]
And works fine as well, renamed all users on outreach now.

Ready to merge.

File: maintenance/scripts/rename_users.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+              "Michael Rosbash",
+              "Carol Greider",
+              "Yoshinori Ohsumi",
+              "Rosalyn Yalow"]
+
+    conn = BlitzGateway(username, password, host=host, port=port)
+    conn.connect()
+    admin_service = conn.getAdminService()
+    try:
+        conn.connect()
```
Comment (jburel) [inline maintenance/scripts/rename_users.py]: duplicate ``conn.connect()`` one outside ``try/finally`` one inside

**Comment by will-moore** [comment]
Removed extra ```conn.connect()```.
NB: The male & females alternate in the list, so user-odd numbers are male and user-even numbers female. Might be useful if you want to give users a matching gender name?

**Comment by jburel** [comment]
I think the script is fine as it is 
Merging

**Pull request merged**